### PR TITLE
do_cmake.sh: use python-3.9 with fedora version 33

### PR DESCRIPTION
**Pull request by sunnyku**
Fedora-33 (rawhide) now has python-3.9.
Build breaks on Fedoa-33 with Traceback:
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
Could NOT find Python3: Found unsuitable version "3.9.0", but required is
exact version "3.8" (found /usr/bin/python3, found components: Interpreter
Development)
    
Fixes: https://tracker.ceph.com/issues/47971
Signed-off-by: Sunny Kumar <sunkumar@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: do_cmake.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ if [ -r /etc/os-release ]; then
       fedora)
           PYBUILD="3.7"
           if [ "$VERSION_ID" -ge "32" ] ; then
-              PYBUILD="3.8"
+              PYBUILD="3"
```
Comment (tchaikov) [inline do_cmake.sh:21]: what if some developer installs python3.9 on fedora31/fedora32? on which python3.9 is not fully supported, and lots of packages are missing.

**Review by tchaikov — CHANGES_REQUESTED** [review]
the title of the commit message reads like a bug report. ideally, the title of the commit message should use the imperative mood. see also https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes . please use `git commit --amend`  and `git push -f` for revising the commit message and its title.

**Comment by smithfarm** [comment]
To follow up on what @tchaikov wrote, the general idea is that what you wrote under "Solution" (in the commit message) should actually be the commit title. The title of the commit should describe what the commit is doing, not what problem it is solving.

     do_cmake: use PYBUILD="3" when fedora version is greater than 32

(where "use" is the "imperative" form of the verb "to use")

You can see examples of commit message titles by doing `git log --oneline --no-merges`

**Comment by sunnyku** [comment]
jenkins test make check

**Review by tchaikov — APPROVED** [review]

**Review by jdurgin — APPROVED** [review]

**Pull request merged**